src/platform/datapath_raw.c

```diff
@@ -423,7 +423,7 @@ _IRQL_requires_max_(DISPATCH_LEVEL)
 QUIC_STATUS
 CxPlatSocketSend(
     _In_ CXPLAT_SOCKET* Socket,
-    _In_ const CXPLAT_ROUTE* Route,
+    _Inout_ CXPLAT_ROUTE* Route,
```

Comment:
I've since realized that this is actually going to need to be passed in and use in `CxPlatSendDataAlloc`. Once we support both "normal" and "accelerated" sockets in parallel, we will need to use different send allocations depending on which socket we use. I'll make the change to move this logic to there instead.